src/community/roadmap/index.md.njk

```diff
@@ -0,0 +1,46 @@
+---
+title: Roadmap
+description: This roadmap shows what we’re working on and planning to do.
+section: Community
+layout: layout-pane.njk
+order: 9
+---
+
+This roadmap shows what we’re working on and planning to do.
+
+Some things on the roadmap might change — the purpose is to tell you what’s coming up and help service teams prepare and plan their own work.
+
+See our [Github team board](https://github.com/orgs/alphagov/projects/4) for more details on our plans and day-to-day activities.
+
+## Recently shipped
+
+- Hosted Design System Day on 21 July 2021
+- Implemented Google Analytics on the Design System website
+- Published checkboxes variant, [none of the above](/components/checkboxes/#add-an-option-for-none-)
+- Reviewed accessibility issue with [conditional reveals](https://accessibility.blog.gov.uk/2021/09/21/an-update-on-the-accessibility-of-conditionally-revealed-questions/)
+- Released [GOV.UK Frontend v3.13.1](https://github.com/alphagov/govuk-frontend/releases/tag/v3.13.1)
+- Published [accessibility acceptance criteria for contributions](https://github.com/alphagov/govuk-frontend/blob/main/docs/contributing/test-components-using-accessibility-acceptance-criteria.md)
+- Completed a mini discovery on how different teams are prototyping across government
+
+## Working on now
+
+- Revise accordion design contributed by GOV.UK to address an accessibility issue
+- Release [GOV.UK Frontend Version 4.0.0](https://github.com/alphagov/govuk-frontend/milestone/22), includes revised accordion, fixes to headers and footers and a number of breaking changes
+- Host community co-design work to [turn task-list into a component](/patterns/task-list-pages/)
+- Host community co-design work to make maps better
+- Improve how [users can find out what we are working on](https://github.com/alphagov/govuk-design-system/issues/1410)
+
+## Coming up next
+
+- Release hide this page component
+- Investigate potential changes to browser support
+- Investigate improving Javascript configuration
+- Improve how contributions are prioritised
+- Ensure the Design System meets WCAG 2.2
+
+## Future plans
+
+- Investigate Sass module system (mid 2022)
+- Investigate removing compatibility mode (mid 2022)
+
+Last updated 27 September 2021
```

Comment:
I commented that it could help to make it more prominent by putting it nearer the top of the page (probs after line 14). Just checking you wanted this to stay here?

Reply:
Done